scripts/generate_rust_analyzer.py

```diff
@@ -65,7 +66,14 @@ def append_crate(display_name, root_module, deps, cfg=[], is_workspace_member=Tr
         [],
         is_proc_macro=True,
     )
-    crates[-1]["proc_macro_dylib_path"] = "rust/libmacros.so"
+
+    libmacros_path = subprocess.run(
+        ["rustc", "--print", "file-names", "--crate-name", "macros", "--crate-type", "proc-macro", "-"],
```

Comment:
This `rustc` is not using `$RUSTC`, is that intended? I guess it is not too important, since you just use it to get the name, but it would be nice to use the one specified by the user.

Reply:
not directly possible, but with import of os. Doing that now

Reply:
Yes, of course, I didn't mean to literally use `$SHELL` in Python, just that we should use the environment variable :)

Reply:
Well I tried. Don't know what exactly python allows :)